spl/src/token.rs

```diff
@@ -28,7 +28,7 @@ pub fn transfer<'a, 'b, 'c, 'info>(
             ctx.accounts.from.clone(),
             ctx.accounts.to.clone(),
             ctx.accounts.authority.clone(),
-            ctx.program.clone(),
+//             ctx.program.clone(),
```

Comment:
Can delete these lines and then merge.

Reply:
Ok, removed these lines. But it's definitely kind of confusing that program id isn't required in the invoke call accounts list.